project/AppOverrides.scala

```diff
@@ -14,18 +14,17 @@
  * limitations under the License.
  */
 
-import LibDependencies.compile
 import sbt._
 
 object AppOverrides {
 
-  def apply(): Set[ModuleID] = overrides
+  def apply(): Seq[ModuleID] = overrides
 
   // we need to override the akka version for now as newer versions are not compatible with reactivemongo
```

Comment:
Do we still need this?

Reply:
I will check if the overrides are still needed for Play 2.6.

Reply:
Rather than overriding in every library, the services which require it can set their overrides in their own project, as documented in the migration guide? If one library has it missing, they will have to explicitly override anyway. (Also a conversation starter to encourage to move to scala 2.12)

Reply:
@colin-lamed removed from the project now.